src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -184,9 +184,13 @@
                               [selectionLimit]="4"
                               (selection)="appSelection()">
             </cd-select-badges>
+            <i *ngIf="data.applications.selected <= 0"
+                i18n-title
+                title="No applications Added (Recommended)"
```

Comment:
```suggestion
                title="No applications added (Recommended)"
```
Let's better use ['sentence case'](https://www.patternfly.org/v4/ux-writing/capitalization/).

Additionally, on the sentence itself, you can find [here some tips on error messages](https://www.patternfly.org/v4/ux-writing/error-messages/): I'd rather lean towards something like "Pools should be associated with an application tag" (which is [the same text as in here](https://docs.ceph.com/en/latest/rados/operations/pools/#associate-pool-to-application), though sounding less mandatory).

Reply:
Agree with that, since its a warning should be less mandatory